README.md

```diff
@@ -44,6 +44,7 @@ If you have many activities, you may find that this script crashes with an "Oper
 - If you're comfortable using Git, just clone the repo from github
 - Otherwise get the latest `zip` (or `tar.gz`) from the [releases page](https://github.com/pe-st/garmin-connect-export/releases)
   and unpack it where it suits you.
+- Install the dependencies: `pip install -r requirements.txt`
```

Comment:
I had success with `python3 -m pip install -r requirements.txt`, but not with `pip install -r requirements.txt`

My python installation only has `/usr/bin/pip3`, no `pip`. I guess using `python3 -m pip` works for more people, what do you think?

Reply:
Yeah, indeed, `pip install -r requirements.txt` depends on where your `pip` executable points to, `python3 -m ` uses the `python3` pip version, which is more likely to work. Will update.